app/routes/onboarding/userDetails.tsx

```diff
@@ -165,69 +169,72 @@ const UserDetails = () => {
+                label="Username"
+                name="username"
+                placeholder="username here"
+                link="hap.day/"
+                value={userForm.username}
+                setValue={updateUsername}
+                err={usernameError() || errors?.username || userErrors.username}
                 handleBlur={handleBlur}
                 handleFocus={handleFocus}
               />
             </div>
-            <div className="basis-3/6 mx-1">
-              <UserInput
-                data-testid="lastname"
-                label="Last name"
-                name="lastname"
-                placeholder="Doe"
-                value={userForm.lastname}
-                setValue={updateLastName}
-                err={errors?.lastname || userErrors.lastname}
-                handleBlur={handleBlur}
-                handleFocus={handleFocus}
-              />
+
+            <div className="flex justify-between ">
+              <div className="basis-3/6 mx-1">
+                <UserInput
+                  data-testid="firstname"
+                  label="First name"
+                  name="firstname"
+                  placeholder="Jane"
+                  value={userForm.firstname}
+                  setValue={updateFirstName}
+                  err={errors?.firstname || userErrors.firstname}
+                  handleBlur={handleBlur}
+                  handleFocus={handleFocus}
+                />
+              </div>
+              <div className="basis-3/6 mx-1">
+                <UserInput
+                  data-testid="lastname"
+                  label="Last name"
+                  name="lastname"
+                  placeholder="Doe"
+                  value={userForm.lastname}
+                  setValue={updateLastName}
+                  err={errors?.lastname || userErrors.lastname}
+                  handleBlur={handleBlur}
+                  handleFocus={handleFocus}
+                />
+              </div>
+            </div>
+            <div>
+              <Dropdown placeholder="select timezone" setUserTimezone={setUserForm} />
             </div>
           </div>
-          <div>
-            <Dropdown placeholder="select timezone" setUserTimezone={setUserForm} />
-          </div>
         </div>
-      </div>
-      <div className=" mx-4 mb-3 basis-1/12">
-        <Button
-          dataTestId="submitButton"
-          label="Submit"
-          size="medium"
-          varient="primary"
-          type={'submit'}
-          style={'my-2'}
-          disabled={![...Object.values(userForm)].every((ele) => ele.length > 0)}
-        />
-      </div>
-    </Form>
+        <div className=" mx-4 mb-3 basis-1/12">
+          <Button
+            dataTestId="submitButton"
+            label="Submit"
+            size="medium"
+            varient="primary"
+            type={'submit'}
+            style={'my-2'}
+            disabled={![...Object.values(userForm)].every((ele) => ele.length > 0)}
```

Comment:
Would it be better to have a variable defining what it does and assign condition check variable here?

Reply:
@harshith-venkatesh , I did not write this. Its previous code just showing up in green due to change in the structure.